crates/requirements-txt/src/lib.rs

```diff
@@ -317,6 +317,27 @@ pub struct RequirementsTxt {
     pub no_index: bool,
 }
 
+/// Calculates column and line based on the cursor and content.
```

Comment:
Assuming you use `unicode-width` per my suggestion below, can you just add a quick note here documenting that we define column in this context as the, "offset according to the visual width of each codepoint."

Reply:
If `unicode-width` is not the right thing to use, then just adding, "offset according to the number of Unicode codepoints."

Reply:
I left it as a comment in 223fd99ac1b44e0557227d1c461a20bf1a9d85f2